addons/account_audit_trail/report/audit_trail_report_views.xml

```diff
@@ -24,13 +23,14 @@
         <field name="mode">primary</field>
         <field name="arch" type="xml">
             <search string="Messages Search">
-                <field name="account_audit_log_move_id"/>
+                <field name="res_id" string="Journal Entry"/>
```

Comment:
We don't have the nice search view anymore
![image](https://github.com/odoo/odoo/assets/44770049/11f8a2b9-2db7-4859-a01d-9a1fd3eaf2dd)

Reply:
Looked at it, and seems more a bug or a missing feature than anything else ... field is in the search view but is not displayed when typing something. Weird. Reference fields are still not correctly supported it seems. Issue is that we are going to make audit trail somehow standard, meaning we won't add a m2o to all possible models that want to use it.

Reply:
This is annoying because that search is important...

But I'm not sure how it would be possible to implement without a dedicated field, because it needs the `name_search` on the right model, which is not really possible if it can come from multiple models.

Reply:
Note that group_by works (which seems more complicated to me compared to a name_search), so I bet something can be done for the search.

Reply:
Actually you can search on it. Using an integer (like 3 -> display invoice 3 audit trail). Definitively looks more like a forgotten feature.

Reply:
Also, if you want to make it standard, have a look at this
https://www.odoo.com/odoo/project/967/tasks/3952457?cids=1

(note that it needs to be merged for the next saas to be able to start the certification process with GoBD)

Reply:
Wow, not sure who did that, but it looks like a really bad idea from security, performance and modular point of view.

Reply:
Yep, you know the models indeed so it is easier in the end :) .

Reply:
Field is back, then it is more a code cleanup commit (a bit less search and stuff like that).